moderators/github.md

```diff
@@ -0,0 +1,55 @@
+# Maintaining Repositories
+
+This page is intended to provide guidelines for maintaining EddieHub's various repositories.
+
+## When To Merge PRs
+
+Pull requests should be merged only after:
+
+- At least 2 maintainers have provided an approving review.
+- All requested changes have been resolved (no "unresolved conversations").
+- All CI checks (GitHub Actions) are passing.
+
+## How to Merge PRs
+
+When merging a pull request, you should generally perform a "Squash and Merge". This will squash all of the commits into one.
+
+When you do this, GitHub's UI will give you a chance to edit the commit message. Be sure to follow the [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) format:
+
+```txt
+<type>(<scope>): <message>
+```
+
+- `type` is the nature of the change:
+  - `feat` for new features
+  - `fix` for bug fixes
+  - `docs` for documentation changes
+  - `chore` for changes such as updating developer configs (`.eslintrc.json`) or similar changes that don't affect the end user.
+- `scope` is **optional**, and explains the part of the codebase the change touches:
+  - `api` for a backend change
+  - `client` for a front end change
+  - `tools` for things like actions
+  - `tests` for adding tests
+  - ...and so on.
+- `message` is a brief description of the changes the pull request makes.
+
+For example, the commit message which created this file was `docs: add drafts for moderation guide`.
+
+Additionally, GitHub will automatically append the pull request number at the end of the default commit message. **DO NOT REMOVE THIS**, as it allows people to reference the pull request that created the commit to check the full commit history.
+
+## Handling Inactive PRs
+
+If a contributor has not responded to reviews or comments on a PR, follow this general timeline for how to respond:
+
+- After 7 days (one week) of inactivity, a maintainer should comment on the PR.
+  - This comment should ping the author, ask for a status update, and remind them that we have a Discord server if they need assistance with contributing.
+- After 14 days (two weeks) of inactivity **since the contributor's last comment**, a maintainer should close the pull request.
```

Comment:
How about configure stale bot it self to do this 🤔

Reply:
I'm personally against stale bot in general, but *especially* for pull requests.

Folks may have become distracted with another project, and the last thing we should do is have a bot send an automated message and dismiss their efforts.